.github/workflows/codeql-analysis.yml

```diff
@@ -0,0 +1,55 @@
+name: "CodeQL"
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
+    paths-ignore:
+      - 'rfd/**'
```

Comment:
If CodeQL doesn't do anything for helm charts, I'd also ignore `charts/**`.

Also, the plugins all use toml for configuration, so you can ignore `**.toml` as well.

Reply:
I'm hesitant to ignore any code changes, as it would require us maintaining all our CodeQL workflows across tons of different repositories. But perhaps. Realistically, we should add https://github.com/helm/chart-testing-action to have proper testing.

TOML seems like it might be supported under Python scanning support in CodeQL -- https://codeql.github.com/docs/codeql-overview/supported-languages-and-frameworks/#python-built-in-support -- would need to double-check for sure.